sale_purchase_requisition/models/purchase_requisition.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+
+
+class PurchaseRequisition(models.Model):
+    _inherit = "purchase.requisition"
+
+    sale_id = fields.Many2one(
+        comodel_name="sale.order",
+        string="Sale Order",
+        help="Sale order related to this purchase requisition.",
+    )
+    sale_user_id = fields.Many2one(
+        comoel_name="res.users",
+        string="Sale Representative",
+        related="sale_id.user_id",
+    )
+    purchase_requisition_sync_warning = fields.Char(compute="_compute_sync_sale_order")
+
+    def _compute_sync_sale_order(self):
+        for rec in self:
+            rec.purchase_requisition_sync_warning = False
+            if rec.sale_id.purchase_requisition_sync_warning:
+                rec.purchase_requisition_sync_warning = _(
+                    "Some lines are not synced on these Sale order:"
+                    + "%s please update this lines manually" % (rec.sale_id.name),
+                )
```

Comment:
This code contains a bug. You should never format the string before passing it to the translation function. It must be done after, always.

Besides, there are a couple of spell errors.

```suggestion
                rec.purchase_requisition_sync_warning = _(
                    "Some lines are not synced on this Sale order: %s. "
                    "Please update these lines manually.",
                    rec.sale_id.name,
                )
```